zingolib/src/wallet/transaction_record.rs

```diff
@@ -171,6 +172,43 @@ impl TransactionRecord {
             self.total_orchard_value_spent,
         ]
     }
+    pub fn get_received_note<D>(
```

Comment:
this fn is extra

Reply:
i mean, it is superceded by the stuff in https://github.com/zingolabs/zingolib/pull/926/files#diff-cc4a46a7c665ac8355eec483ff4eb95fa7b6f50445e53bdfdd29414bec4233ac and wont be implemented